src/index.js

```diff
@@ -2,16 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import store from './store/index.js'
+import { Provider } from 'react-redux';
```

Comment:
Libraries should be imported before the local files:

```javascript
import { Provider } from 'react-redux';
import ReactDOM from 'react-dom/client';

import App from './App';
import store from './store/index.js';
import './index.css';
```

**Use semicolons** in the end of every line!